docs/src/main/sphinx/functions/comparison.rst

```diff
@@ -214,3 +214,31 @@ you need to match the used escape character as well, you can escape it.
 
 If you want to match for the chosen escape character, you simply escape itself.
 For example, you can use ``\\`` to match for ''\''.
+
+Row comparison: IN
+------------------------
+
+The ``IN`` operator can be use to fetch multiple rows according to mutiple values specified in WHERE clause::
+
+    SELECT * FROM region WHERE name IN ('AMERICA', 'AFRICA');
+    
+It is shorthand of mutiple ``OR`` conditions::
+    
+    SELECT * FROM region WHERE name = 'AMERICA' OR name = 'AFRICA';
+    
+It will return null if there are no rows matching the ``IN`` condition and hence the query will be null.
```

Comment:
I don't think that is how nulls work or a null from a where clause.  `IN` can return `true`, `false`, or `null`.   It returns `true` if the value is in the list, `false` if not in the list and the list contains no `null` values, and `null` if the value is not in the list and the value contains a `null`.  A null in a `WHERE` is really orthogonal to the `IN` documentation, but a row is only accepted if the `WHERE` evaluates to `true`.